### PR TITLE
Remove redundant word

### PR DESCRIPTION
**Pull request by taylortrimble**
I think the word "that" before "to" (see diff) is redundant.

### CODE REVIEW

File: windows-apps-src/design/controls-and-patterns/forms.md [hunk 1]
```diff
@@ -239,4 +239,4 @@ To learn how to connect form input to an **Azure** database and see a fully impl
 
 ## Related topics
 - [Input controls](controls-and-events-intro.md)
-- [Typography](../style/typography.md)
\ No newline at end of file
+- [Typography](../style/typography.md)
```
Comment (taylortrimble) [inline windows-apps-src/design/controls-and-patterns/forms.md:242]: GitHub's online editor added this, since there was no EOF newline.

**Comment by mikematteson** [comment]
@serenaz  : PR **passed** acceptance checklist. Please review in case you want to correct this differently ("that displays" instead of "to display"?). Approve/merge if appropriate. Thanks!

**Comment by taylortrimble** [comment]
I've made an alternative PR for you that uses "that displays" instead of "to display". #583.

**Review by serenaz — APPROVED** [review]
Thanks for catching the typo!

**Pull request merged**